packaging/opensuse/snapd.spec

```diff
@@ -171,69 +184,68 @@ make %{?_smp_mflags} -C cmd check
 # TODO: instead of removing it move this to a dedicated golang package
 rm -rf %{buildroot}%{_libexecdir}64/go
 rm -rf %{buildroot}%{_libexecdir}/go
-find %{buildroot}
 # Move snapd, snap-exec, snap-seccomp and snap-update-ns into %{_libexecdir}/snapd
 install -m 755 -d %{buildroot}%{_libexecdir}/snapd
-mv %{buildroot}/usr/bin/snapd %{buildroot}%{_libexecdir}/snapd/snapd
-mv %{buildroot}/usr/bin/snap-exec %{buildroot}%{_libexecdir}/snapd/snap-exec
-mv %{buildroot}/usr/bin/snap-update-ns %{buildroot}%{_libexecdir}/snapd/snap-update-ns
-mv %{buildroot}/usr/bin/snap-seccomp %{buildroot}%{_libexecdir}/snapd/snap-seccomp
+mv %{buildroot}%{_bindir}/snapd %{buildroot}%{_libexecdir}/snapd/snapd
+mv %{buildroot}%{_bindir}/snap-exec %{buildroot}%{_libexecdir}/snapd/snap-exec
+mv %{buildroot}%{_bindir}/snap-update-ns %{buildroot}%{_libexecdir}/snapd/snap-update-ns
+mv %{buildroot}%{_bindir}/snap-seccomp %{buildroot}%{_libexecdir}/snapd/snap-seccomp
 # Install profile.d-based PATH integration for /snap/bin
 #   and XDG_DATA_DIRS for /var/lib/snapd/desktop
 make -C data/env install DESTDIR=%{buildroot}
 
 # Generate and install man page for snap command
 install -m 755 -d %{buildroot}%{_mandir}/man1
-%{buildroot}/usr/bin/snap help --man >  %{buildroot}%{_mandir}/man1/snap.1
+%{buildroot}%{_bindir}/snap help --man >  %{buildroot}%{_mandir}/man1/snap.1
 
 # TODO: enable gosrc
 # TODO: enable gofilelist
 
 # Install all the C executables
-%{make_install} -C cmd
+%make_install -C cmd
 # Undo special permissions of the void directory
-chmod 755 %{?buildroot}/var/lib/snapd/void
+chmod 755 %{buildroot}%{_sharedstatedir}/snapd/void
 # Remove traces of ubuntu-core-launcher. It is a phased-out executable that is
 # still partially present in the tree but should be removed in the subsequent
 # release.
-rm -f %{?buildroot}/usr/bin/ubuntu-core-launcher
+rm -f %{buildroot}%{_bindir}/ubuntu-core-launcher
 # NOTE: we don't want to ship system-shutdown helper, it is just a helper on
 # ubuntu-core systems that exclusively use snaps. It is used during the
 # shutdown process and thus can be left out of the distribution package.
-rm -f %{?buildroot}%{_libexecdir}/snapd/system-shutdown
+rm -f %{buildroot}%{_libexecdir}/snapd/system-shutdown
 # Install the directories that snapd creates by itself so that they can be a part of the package
-install -d %buildroot/var/lib/snapd/{assertions,desktop/applications,device,hostfs,mount,apparmor/profiles,seccomp/bpf,snaps}
+install -d %{buildroot}%{_sharedstatedir}/snapd/{assertions,desktop/applications,device,hostfs,mount,apparmor/profiles,seccomp/bpf,snaps}
 
-install -d %buildroot/var/lib/snapd/{lib/gl,lib/gl32,lib/vulkan}
-install -d %buildroot/var/cache/snapd
-install -d %buildroot/snap/bin
+install -d %{buildroot}%{_sharedstatedir}/snapd/{lib/gl,lib/gl32,lib/vulkan}
+install -d %{buildroot}%{_localstatedir}/cache/snapd
+install -d %{buildroot}/snap/bin
```

Comment:
I'd be interesting if we had `%{snapdir}`

Reply:
I can define a macro for it, which would probably be a good idea anyway, I suppose...